lib/node/http.js

```diff
@@ -378,6 +378,8 @@ class HTTP extends Server {
       const height = valid.u32('height');
 
       enforce(height != null, 'Height is required.');
+      enforce(height <= this.chain.tip,
```

Comment:
Should be `this.chain.height` or `this.chain.tip.height`.

Reply:
Ah my fault, I think I accidentally dropped that on a formatting change. Should be fixed now, also switched to this.chain.height as that seemed to be used more than tip.height.